integration/config/configtx.yaml

```diff
@@ -259,7 +259,15 @@ Orderer: &OrdererDefaults
 
     # Orderer Type: The orderer implementation to start.
     # Available types are "solo" and "kafka".
-    OrdererType: solo
+    OrdererType: etcdraft
+    EtcdRaft:
+        Consenters:
+        - Host: orderer.example.com
+          Port: 7059
```

Comment:
what about this comment? I don't think it's addressed?  Comments of course can be overruled. However, in that case it would be nice if you could add a comment here why not to addresse it. See also other comment that the reviewer should declare a comment resolved, not the PR owner.